esmvaltool/diag_scripts/radiation_budget/radiation_budget.py

```diff
@@ -0,0 +1,468 @@
+"""Plot the global radiation budget."""
+
+# To run the doctests:
+# % cd ESMValTool/esmvaltool/
+# % python -m doctest diag_scripts/radiation_budget/radiation_budget.py
+import logging
+import os
+
+import iris
+import matplotlib.pyplot as plt
+import numpy as np
+import yaml
+from esmvalcore.iris_helpers import var_name_constraint
+
+from esmvaltool.diag_scripts.shared import (
+    group_metadata,
+    run_diagnostic,
+    save_figure,
+)
```

Comment:
A very minor suggestion to alphabetise imports:
```suggestion

from esmvaltool.diag_scripts.shared import (
    group_metadata,
    run_diagnostic,
    save_figure,
)
from esmvalcore.iris_helpers import var_name_constraint
```